Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h

```diff
@@ -206,6 +207,11 @@ class AODProducerWorkflowDPL : public Task
   int64_t mTFNumber{-1};
   int mTruncate{1};
   int mRecoOnly{0};
+  TString mResFile{"AO2D"};
+  TString mLPMProdTag{"LHC21Axx"};
+  TString mAnchorPass{"pass1"};
+  TString mAnchorProd{"LHC15o"};
+  TString mRecoPass{"pass1"};
```

Comment:
Please put empty defaults. We should not have meaningful defaults (like it was for the run number), this causes trouble often. ;)